modules/apps/commerce/commerce-product-test/src/testFunctional/macros/CommerceChannels.macro

```diff
@@ -22,4 +22,41 @@ definition {
 		Click(locator1 = "Icon#CLOSE");
 	}
 
+	macro paymentConfiguration {
+		CommerceEntry.clickTableEntryEditButton(
+			entryName = "${paymentMethod}",
+			table = "Payment Methods");
+
+		SelectFrame.selectFrameNoLoading(locator1 = "CommerceEntry#IFRAME_SIDE_PANEL");
+
+		Click(
+			navItem = "Configuration",
+			locator1 = "CommerceEntry#PAYMENT_METHODS_SIDE_PANEL_NAV_ITEMS");
+
+		Type(
+			locator1 = "TextInput#ANY",
+			key_text = "Client ID",
+			value1 = "PayPal Client ID");
+
+		Type(
+			locator1 = "TextInput#ANY",
+			key_text = "Client Secret",
+			value1 = "PayPal Client Secret");
+
+		Select(
+				key_fieldLabel = "Mode",
+				locator1 = "Select#GENERIC_SELECT_FIELD",
+				value1 = "Sandbox");
```

Comment:
Can we use a variable? in this way we can use this macro in the future not only for Sandbox mode

Reply:
Fixed.